README.md

```diff
@@ -82,10 +82,20 @@ tests/test-helpers.js:
 import { forceModulesToBeLoaded, sendCoverage } from 'ember-cli-code-coverage/test-support';
 import Qunit from 'qunit';
 
-QUnit.done(async function() {
-  forceModulesToBeLoaded();
-  await sendCoverage();
-});
+if (config.APP.isRunningWithServerArgs) {
```

Comment:
The README would also need to indicate where to set this

Reply:
Hi Kate, thanks for catching that.  I have added another section above test-helpers.js in the README to callout the config setting being leveraged.